pyk/src/pyk/ktool/kprove.py

```diff
@@ -255,19 +255,19 @@ def prove_claim(
                 depth=depth,
             )
 
-    def get_claim_modules(
+    def parse_modules(
```

Comment:
Is the module parser exposed by `kprove` general enough to parse an arbitrary module?

Reply:
Hmmmm, I _think_ it should be? But I'm not 100% on that.